src/wdio/services/TerraCommands/accessibility.js

```diff
@@ -1,29 +1,39 @@
 import determineOptions from './determine-test-options';
 
 /**
- * The actual it block for running axe and validating no accessibility violations were found.
+ * The run axe to validate no accessibility violations were found.
  * @param {Object} options - The Axe options. Available options are viewports and
  * rules.
  */
-const accessibleItBlock = (options) => {
-  global.it('is accessible', () => {
-    const axeResults = global.browser.axe(options);
-    global.expect(axeResults).to.be.accessible();
-  });
+const runAccessibilityTest = (options) => {
+  const axeResults = global.browser.axe(options);
+  global.expect(axeResults).to.be.accessible();
 };
 
 /**
 * A mocha-chai convenience test case to assert accessibility.
 * @param {Object} options - The Axe options. Available options are viewports,
 * rules, and context. See https://www.axe-core.org/docs/.
 */
-const beAccessible = (...args) => {
-  accessibleItBlock(determineOptions.axeOptions(args));
+const validatesAccessibility = (...args) => {
+  runAccessibilityTest(determineOptions.axeOptions(args));
```

Comment:
Instead of having both functions call runAccessibilityTest, you could have `itIsAccessible` call `validatesAccessibility`

```javascript
const validatesAccessibility = (...args) => {
  const axeResults = global.browser.axe(determineOptions.axeOptions(args));
  global.expect(axeResults).to.be.accessible();
};

const itIsAccessible = (args) => {
  global.it('is accessible', () => {
    runAccessibilityTest(args);
  });
};

```

Reply:
Talked offline, we need to keep as is for people using `Terra.validates.element` such the validates.element ignores viewports configuration